package.json

```diff
@@ -87,6 +87,7 @@
     "react-flatpickr": "^3.10.7",
     "react-measure": "^2.5.2",
     "react-monaco-editor": "^0.47.0",
+    "react-number-format": "^5.3.1",
```

Comment:
Since the package already handles the input pattern matching, value conversion (which basically strips out the , separators) and formatting all in one without having to do all those manually on each input change.